knowledge/music/events/cochella_2023/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Ana Foubert
+domain: music
+seed_examples:
+- answer: |SG Lewis
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!